python/src/lib.rs

```diff
@@ -297,19 +298,32 @@ fn json_value_to_py(value: &serde_json::Value, py: Python) -> PyObject {
 /// skipped during a scan.
 fn filestats_to_expression<'py>(
     py: Python<'py>,
+    schema: &ArrowSchema,
     partitions_values: &HashMap<String, Option<String>>,
     stats: Option<Stats>,
 ) -> PyResult<Option<&'py PyAny>> {
     let ds = PyModule::import(py, "pyarrow.dataset")?;
     let field = ds.getattr("field")?;
+    let pa = PyModule::import(py, "pyarrow")?;
     let mut expressions: Vec<PyResult<&PyAny>> = Vec::new();
 
     for (column, value) in partitions_values.iter() {
         if let Some(value) = value {
+            // value is a string, but needs to be parsed into appropriate type
+            let column_type = schema
+                .field_with_name(column)
+                .map_err(|_| PyDeltaTableError::new_err("Partition column not found in schema"))?
+                .data_type()
+                .clone()
+                .into_py(py);
+            // pa.scalar(value).cast(type)
```

Comment:
Is it a missing comment?

Reply:
It's meant to annotate what the next line is doing in Python. I can add `Python for below: ` prefix to it for clarity.

Reply:
I ended up just deleting that comment.